VisualStudio.gitignore

```diff
@@ -259,15 +259,20 @@ ServiceFabricBackup/
 *.ldf
 *.ndf
 
-# Business Intelligence projects
+# SQL Server Integration Services
+*.dtproj.user
```

Comment:
This should be unnecessary given the `*.user` rule on line 9, unless I'm missing some context.

Reply:
Somehow, my project need this line to work.

Reply:
Can you confirm that this isn't needed? `*.user` should also handle `*.dtproj.user` based on this quick test I did locally:

```shellsession
$ git init test-user
Initialized empty Git repository in /Users/shiftkey/src/test/test-user/.git/
$ cd test-user
$ touch foo.user
$ touch foo.dtproj.user
$ touch foo
$ git status
On branch master

No commits yet

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	foo
	foo.dtproj.user
	foo.user

nothing added to commit but untracked files present (use "git add" to track)
$ echo "*.user" >> .gitignore
$ git  status
On branch master

No commits yet

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.gitignore
	foo

nothing added to commit but untracked files present (use "git add" to track)
```

If we don't need this rule, we can drop this section to keep this template a bit neater.